haystack_experimental/components/agents/agent.py

```diff
@@ -232,11 +246,17 @@ def _load_component(component_data: Dict[str, Any]) -> Component:
 
         return instance
 
-    def run(self, messages: List[ChatMessage], **kwargs) -> Dict[str, Any]:
+    def run(
+            self,
+            messages: List[ChatMessage],
+            streaming_callback: Optional[SyncStreamingCallbackT] = None,
+            **kwargs
```

Comment:
small nit on indenting
```suggestion
        self,
        messages: List[ChatMessage],
        streaming_callback: Optional[SyncStreamingCallbackT] = None,
        **kwargs
```